app/components/post_list/post/body/reactions/reactions.tsx

```diff
@@ -121,18 +130,26 @@ const Reactions = ({currentUserId, canAddReaction, canRemoveReaction, disabled,
         }
 
         pressed.current = false;
-    };
+    }, [canRemoveReaction, canAddReaction, disabled]);
 
-    const showReactionList = () => {
-        const screen = 'ReactionList';
+    const showReactionList = useCallback((initialEmoji: string) => {
+        const screen = Screens.REACTIONS;
         const passProps = {
+            initialEmoji,
             postId,
         };
 
+        Keyboard.dismiss();
+        const title = isTablet ? intl.formatMessage({id: 'post.reactions.title', defaultMessage: 'Reactions'}) : '';
+
         if (!pressed.current) {
-            showModalOverCurrentContext(screen, passProps);
+            if (isTablet) {
+                showModal(screen, title, passProps, bottomSheetModalOptions(theme, 'close-post-reactions'));
+            } else {
+                showModalOverCurrentContext(screen, passProps);
+            }
         }
-    };
+    }, [intl, postId, theme]);
```

Comment:
Probably you should depend also on "isTablet" (may change if you get to split view, right?)